picodom.d.ts

```diff
@@ -0,0 +1,26 @@
+export interface VNode<D> {
+  tag: string
+  data?: D
+  children: VElement<{}>[]
+}
+
+export interface Component<D> {
+  (data?: D, ...children: VElement<{}>[]): VNode<D>
+}
+
+export type VElement<D> = VNode<D> | string
+
+export function h<D>(
+  tag: Component<D> | string,
+  data?: D,
```

Comment:
Can we rename `D` to something more meaningful?

- Data
- DataAttributes
- VNodeData
- VNodeAttributes